forest/src/cli/mod.rs

```diff
@@ -186,6 +186,9 @@ pub struct CliOpts {
     /// Daemonize Forest process
     #[structopt(long)]
     pub detach: bool,
+    /// Automatically download the latest snapshot from a trusted source
+    #[structopt(long, short)]
```

Comment:
I'd omit `short` here. `-d` seems more suitable for `detach`.

Reply:
Yup, that was an oversight. I'll change it so they don't clash